MapboxNavigation/EndOfRouteViewController.swift

```diff
@@ -143,19 +144,18 @@ class EndOfRouteViewController: UIViewController {
     }
     
     private func updateInterface() {
-        primary.text = string(for: destination)
+        guard let name = destination?.name?.nonEmptyString else { return styleForUnnamedDestination() }
+        primary.text = name
     }
 
     private func clearInterface() {
         primary.text = nil
         stars.rating = 0
     }
     
-    private func string(for destination: Waypoint?) -> String {
-        guard let destination = destination else { return "Unknown" }
-        guard destination.name?.isEmpty ?? false else { return destination.name! }
-        let coord = destination.coordinate
-        return String(format: "%.2f", coord.latitude) + "," + String(format: "%.2f", coord.longitude)
+    private func styleForUnnamedDestination() {
+        staticYouHaveArrived.alpha = 0.0
+        primary.text = NSLocalizedString("You have arrived", bundle: .mapboxNavigation, comment:"")
```

Comment:
@JThramer, this string isn’t actually localizable. See other calls to `NSLocalizedString()` in this codebase for examples. To wit: the first argument should be a unique string ID, typically in UPPER_SNAKE_CASE, and `value` should be set to the English default. Then you’ll need to add an entry to the relevant .strings file. Until that happens, Transifex will be unable to pick up this string, because it only looks at .strings files, not code.